pymemcache/client/base.py

```diff
@@ -744,6 +750,10 @@ def _fetch_cmd(self, name, keys, expect_cas):
                 elif name == b'stats' and line.startswith(b'STAT'):
                     key_value = line.split()
                     result[key_value[1]] = key_value[2]
+                elif name == b'stats' and line.startswith(b'ITEM'):
+                    # For 'stats cachedump' commands
+                    key_value = line.split()
```

Comment:
I think you could also use `line.split(b' ', 1)` here so you don't need to re-join the remainder below.